packages/paste-core/components/breadcrumb/src/index.tsx

```diff
@@ -14,39 +15,51 @@ const BreadcrumbSeparator: React.FC = () => (
     paddingLeft="space20"
     paddingRight="space20"
     role="presentation"
+    element={`${element}_SEPARATOR`}
   >
     /
   </Text>
 );
 
 export interface BreadcrumbItemProps extends React.HTMLAttributes<HTMLLIElement> {
   children: NonNullable<React.ReactNode>;
+  element?: BoxElementProps['element'];
   href?: string;
   last?: boolean;
 }
 
 const BreadcrumbItem = React.forwardRef<HTMLAnchorElement, BreadcrumbItemProps>(
-  ({children, href, last, ...props}, ref) => {
+  ({children, element, href, last, ...props}, ref) => {
+    const separatorElementName = element || 'BREADCRUMB';
```

Comment:
I'd recommend doing this inline on line 62 like this:
```{!last && <BreadcrumbSeparator element={`${element}_SEPARATOR`} />}```
The same way ANCHOR and TEXT are being handled

Reply:
Sure, I can make that change. Curious if this is a preference in style or if there is a functional benefit? I've gotten in the habit of declaring constants for these kinds of things rather than using inline string interpolation when there are conditionals since it provides a bit more clarity on specifically why the interpolation is there. I know there is technically memory spent on declaring the constant, but I think it's pretty minimal.

Reply:
The closer the variables are to where they are used, without sacrificing readability, the better generally.
1- Avoids a const declaration, which lets readers of this code not need to figure out what this line does as they read through the code.
2- Since line 62 is conditional, `{!last && ...` then the string concat doesn't need to run if it isn't used. Using locality this way provides immediate benefits.
3- We do it on lines 47 and 54 :P